<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SluaDebugger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="DebugConsole" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 170, 0);
font: 12pt &quot;Arial&quot;;</string>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="backgroundVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DebugConsole</class>
   <extends>QPlainTextEdit</extends>
   <header>debugconsole.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
